docs/configuration/smime_email_signing.rst

```diff
@@ -0,0 +1,298 @@
+
+- A **public certificate** (`.crt` file) that verifies your identity
+- A **private key** (`.pem` file) that signs the Emails
+
+Self-signed certificates
+=========================
+
+For testing purposes, you can create self-signed certificates. However, for production use, you should obtain certificates from a trusted Certificate Authority.
+
+To create a self-signed certificate and private key:
+
+.. code-block:: bash
+
+    # Create a private key and certificate signing request
+    openssl req -newkey rsa:4096 -nodes -keyout sender@example.com.pem -out sender@example.com.csr
+    
+    # Create a self-signed certificate valid for 1 year
+    openssl x509 -req -days 365 -in sender@example.com.csr -signkey sender@example.com.pem -out sender@example.com.crt
+
+.. important::
+    Replace ``sender@example.com`` with the actual Email address you're using to send Emails from Mautic.
+
+Production certificates
+=======================
+
+For production use, obtain S/MIME certificates from a trusted Certificate Authority - CA. Many Certificate Authorities offer S/MIME certificates, and the process typically involves:
+
+1. Generating a Certificate Signing Request - CSR
+2. Submitting the CSR to the CA along with identity verification documents
+3. Receiving the signed certificate from the CA
+
+For detailed instructions on obtaining production certificates, see the :xref:`SparkPost S/MIME Technical Guide`.
+
+Installing certificates
+************************
+
+Certificate file naming
+=======================
+
+Name certificates according to the Email address they're for:
+
+- Public certificate: ``email@example.com.crt``
+- Private key: ``email@example.com.pem`` - plain text - or ``email@example.com.pem.enc`` - encrypted
+
+Replace ``email@example.com`` with the actual sender Email address.
+
+.. important::
+    The Email address in the filename must exactly match the **From** address used when sending Emails.
+
+Certificate directory structure
+================================
+
+Place your certificate files in the directory specified by ``smime_certificates_path``:
+
+.. code-block:: text
+
+    /var/smime_certificates/
+    ├── admin@example.com.crt
+    ├── admin@example.com.pem
+    ├── support@example.com.crt
+    └── support@example.com.pem
+
+Setting permissions
+===================
+
+Ensure that the web server user has read access to the certificate directory and files:
+
+.. code-block:: bash
+
+    # Set ownership (replace www-data with your web server user)
+    chown -R www-data:www-data /path/to/mautic/var/smime_certificates
+    
+    # Set directory permissions
+    chmod 755 /path/to/mautic/var/smime_certificates
+    
+    # Set certificate permissions
+    chmod 644 /path/to/mautic/var/smime_certificates/*.crt
+    chmod 600 /path/to/mautic/var/smime_certificates/*.pem
+
+.. warning::
+    Private keys (`.pem` files) should have restrictive permissions (``600``) to prevent unauthorized access.
```

Comment:
Add backticks for inline code:


```suggestion
    Private keys (``.pem`` files) should have restrictive permissions (``600``) to prevent unauthorized access.
```